ts/utils/share.ts

```diff
@@ -21,3 +22,13 @@ export const shareBase64Content = (
     errorMsg => new Error(String(errorMsg))
   );
 };
+
+/***
+ * Return true if the share is available
```

Comment:
@fabriziofff 
this PR lgtm
If you want we can leave a more detail about disabled sharing
```suggestion
 * Return true if the share is available
 * sharing is disabled on Android versions older than 21 due to a bug causing crash (see https://www.pivotaltracker.com/n/projects/2048617/stories/174295714)
```